src/styles/blocks/catalog.scss

```
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, minmax($card-size, 1fr));
+  justify-content: center;
+  justify-items: center;
+  gap: $gap;
+
+  @media (min-width: 488px) {
+    & {
```

```suggestion
    & {
```

no need to use nesting here

---

you can just leave
```
grid-template-columns:...